tests/test_calendar.py

```diff
@@ -28,3 +29,39 @@ def test_calendar_event():
     assert cal_str.startswith('BEGIN:VCALENDAR')
     assert 'SUMMARY:Earth Day' in cal_str
     assert cal_str.endswith('END:VCALENDAR')
+
+def test_calendar_default_timezone():
```

Comment:
Ah, I see this here now; that's a nice pattern, perhaps we should do more of these, instead of having the example files.